src/scripts.js

```diff
@@ -85,3 +122,37 @@ function addNewBooking(event) {
     }
 
   }
+
+function logInUser() {
+  let username = usernameField.value
+  let password = passwordField.value
+  let customerID;
+
+  if(username.startsWith('customer') && password === 'overlook2021'){
+     customerID = parseInt(username.split('customer')[1])
+  }
+  verifyUser(customerID, password);
+
+}
+
+function verifyUser(customerID, password){
+  guestData.find(guest => guest.id === customerID)
+  if (!guestData.find(guest => guest.id === customerID) ||
+    password !== 'overlook2021') {
+    domUpdates.displaySignInError();
```

Comment:
do the form fields empty out after an unsuccessful sign in? What if the fields are empty and someone trys to log in?